setup.py

```diff
@@ -49,7 +49,7 @@
         [console_scripts]
         metaflow=metaflow.cmd.main_cli:start
       """,
-    install_requires=["requests", "boto3"],
+    install_requires=["requests", "boto3", "pyyaml"],
```

Comment:
Ok, I decided to output all of the templates to a directory (using json), instead of pyyaml... @savingoyal WDYT?

Reply:
this would break the existing users. we could introduce new options - `--only-workflow-template-json`, `--only-event-sensor-json`, `--only-cron-workflow-template-json` etc. where `--only-json` is aliased to `--only-workflow-template-json`.

Reply:
ah - `--only-json` is not replaced by `--only-json-directory`. i misread your comment - my apologies

Reply:
Ok, just getting back around to this. I've made the changes outlined here https://github.com/Netflix/metaflow/pull/1947#discussion_r1715434964... PTAL!